jwst/datamodels/schemas/keyword_msatargacq.schema.yaml

```diff
@@ -0,0 +1,120 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/keyword_msatargacq.schema"
+title: MSA target acquisition data model's additional keywords
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      sam_roll:
+        title: "Calculated small angle maneuver roll comp (arcsec)"
```

Comment:
The convention for FITS keyword comments/titles that include units is to have the units appear in square brackets at the very beginning of the comment string, e.g. "[arcsec] Calculated small angle maneuver roll comp". This formatting should be applied to all keyword comments that indicate units.

Reply:
I'm making those revisions.